crates/sui-core/src/transaction_input_checker.rs

```diff
@@ -256,10 +256,8 @@ fn check_one_object(
                 !object.is_package(),
                 SuiError::MovePackageAsObject { object_id }
             );
-            // wrapped objects that are then deleted will be set to MAX,
```

Comment:
I couldn't spot a place where wrapped-and-then-deleted objects get this treatment, is this something that still happens?

Reply:
I think this was out of date when parent_sync was used for this (but it will also go away with lamport time stamps)